quick-start-guide/src/main/resources/bin/qsg.sh

```diff
@@ -1864,6 +1841,38 @@ echo "<soapenv:Envelope xmlns:soapenv="\"http://schemas.xmlsoap.org/soap/envelop
                     <xsd1:alwaysSendMappedLocalSubjectId>false</xsd1:alwaysSendMappedLocalSubjectId>
                     <!--Optional:-->
                     <xsd1:localClaimDialect>true</xsd1:localClaimDialect>
+                     <xsd1:claimMappings>
```

Comment:
@omindu  Here is the SOAP message 

<<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://org.apache.axis2/xsd" xmlns:xsd1="http://model.common.application.identity.carbon.wso2.org/xsd">
    <soapenv:Header/>
    <soapenv:Body>
        <xsd:updateApplication>
            <!--Optional:-->
            <xsd:serviceProvider>
                <!--Optional:-->
                <xsd1:applicationID>354</xsd1:applicationID>
                <!--Optional:-->
                <xsd1:applicationName>dispatch</xsd1:applicationName>
                <!--Optional:-->
                <xsd1:claimConfig>
                    <!--Optional:-->
                    <xsd1:alwaysSendMappedLocalSubjectId>false</xsd1:alwaysSendMappedLocalSubjectId>
                    <!--Optional:-->
                    <xsd1:localClaimDialect>true</xsd1:localClaimDialect>
                    <xsd1:claimMappings>
	                  <!--Optional:-->
	                  <xsd1:localClaim>
	                     <!--Optional:-->
	                     <xsd1:claimUri>http://wso2.org/claims/fullname</xsd1:claimUri>
	                  </xsd1:localClaim>
	                  <!--Optional:-->
	                  <xsd1:mandatory>true</xsd1:mandatory>
	                  <!--Optional:-->
	                  <xsd1:remoteClaim>
	                     <!--Optional:-->
	                     <xsd1:claimUri>http://wso2.org/claims/fullname</xsd1:claimUri>
	                  </xsd1:remoteClaim>
	                  <!--Optional:-->
	                  <xsd1:requested>true</xsd1:requested>
            		</xsd1:claimMappings>
            		
            		<xsd1:claimMappings>
	                  <!--Optional:-->
	                  <xsd1:localClaim>
	                     <!--Optional:-->
	                     <xsd1:claimUri>http://wso2.org/claims/emailaddress</xsd1:claimUri>
	                  </xsd1:localClaim>
	                  <!--Optional:-->
	                  <xsd1:mandatory>true</xsd1:mandatory>
	                  <!--Optional:-->
	                  <xsd1:remoteClaim>
	                     <!--Optional:-->
	                     <xsd1:claimUri>http://wso2.org/claims/emailaddress</xsd1:claimUri>
	                  </xsd1:remoteClaim>
	                  <!--Optional:-->
	                  <xsd1:requested>true</xsd1:requested>
            		</xsd1:claimMappings>

                </xsd1:claimConfig>
                <!--Optional:-->
                <xsd1:description>sample service provider</xsd1:description>
                <!--Optional:-->
                <xsd1:inboundAuthenticationConfig>
                    <!--Zero or more repetitions:-->
                    <xsd1:inboundAuthenticationRequestConfigs>
                        <!--Optional:-->
                        <xsd1:inboundAuthKey>saml2-web-app-pickup-dispatch.com</xsd1:inboundAuthKey>
                        <!--Optional:-->
                        <xsd1:inboundAuthType>samlsso</xsd1:inboundAuthType>
                        <!--Zero or more repetitions:-->
                        <xsd1:properties>
                            <!--Optional:-->
                            <xsd1:name>attrConsumServiceIndex</xsd1:name>
                            <!--Optional:-->
                            <xsd1:value>1223160755</xsd1:value>
                        </xsd1:properties>
                    </xsd1:inboundAuthenticationRequestConfigs>
                </xsd1:inboundAuthenticationConfig>
                <!--Optional:-->
                <xsd1:inboundProvisioningConfig>
                    <!--Optional:-->
                    <xsd1:provisioningEnabled>false</xsd1:provisioningEnabled>
                    <!--Optional:-->
                    <xsd1:provisioningUserStore>PRIMARY</xsd1:provisioningUserStore>
                </xsd1:inboundProvisioningConfig>
                <!--Optional:-->
                <xsd1:localAndOutBoundAuthenticationConfig>
                    <!--Optional:-->
                    <xsd1:alwaysSendBackAuthenticatedListOfIdPs>false</xsd1:alwaysSendBackAuthenticatedListOfIdPs>
                    <!--Optional:-->
                    <xsd1:authenticationStepForAttributes></xsd1:authenticationStepForAttributes>
                    <!--Optional:-->
                    <xsd1:authenticationStepForSubject></xsd1:authenticationStepForSubject>
                    <xsd1:authenticationType>default</xsd1:authenticationType>
                    <!--Optional:-->
                    <xsd1:subjectClaimUri>http://wso2.org/claims/fullname</xsd1:subjectClaimUri>
                </xsd1:localAndOutBoundAuthenticationConfig>
                <!--Optional:-->
                <xsd1:outboundProvisioningConfig>
                    <!--Zero or more repetitions:-->
                    <xsd1:provisionByRoleList></xsd1:provisionByRoleList>
                </xsd1:outboundProvisioningConfig>
                <!--Optional:-->
                <xsd1:permissionAndRoleConfig></xsd1:permissionAndRoleConfig>
                <!--Optional:-->
                <xsd1:saasApp>false</xsd1:saasApp>
            </xsd:serviceProvider>
        </xsd:updateApplication>
    </soapenv:Body>
</soapenv:Envelope>